app/helpers/questions_helper.rb

```diff
@@ -46,7 +46,7 @@ def last_question
   end
 
   def first_question_seen?
-    session[:urgent_medical_help].present?
+    session[:need_help_with].present?
```

Comment:
The `first_question_seen?` method can be completely removed, since it was only used in `ApplicationController`, but is no longer being called.

Reply:
I've removed this - thanks Bruce!